### PR TITLE
Add up down shortcuts

### PR DESCRIPTION
**Pull request by Jolonauh**

### CODE REVIEW

File: src/main/java/seedu/address/history/CommandState.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package seedu.address.history;
+
+/**
+ * placeholder
```
Comment (Rishit02) [inline src/main/java/seedu/address/history/CommandState.java:4]: Can update the javadocs

File: src/main/java/seedu/address/history/HistoryManager.java [hunk 2]
```diff
@@ -48,10 +50,12 @@ public void rollBackState() throws HistoryException {
      */
     @Override
     public void rollForwardState() throws HistoryException {
-        if (currStateIdx == modelStates.size() - 1) {
+        int boundary = hasBuffer ? states.size() - 2 : states.size() - 1;
+        if (currStateIdx >= states.size() - 1) {
             throw new HistoryException("You can't roll forward the state anymore!");
         }
         currStateIdx += 1;
+        System.out.println(currStateIdx);
```
Comment (Rishit02) [inline src/main/java/seedu/address/history/HistoryManager.java:58]: Can remove this

File: src/main/java/seedu/address/ui/CommandBox.java [hunk 3]
```diff
@@ -46,21 +47,29 @@ public CommandBox(CommandExecutor commandExecutor, NextCommandRetriever nextComm
...
-            break;
+        try {
+            switch (event.getCode()) {
+            case UP:
+                System.out.println("Up key pressed");
+                commandTextField.setText(previousCommandRetriever.retrievePreviousCommand());
+                break;
+            case DOWN:
+                try {
+                    System.out.println("Down key pressed");
```
Comment (Rishit02) [inline src/main/java/seedu/address/ui/CommandBox.java:58]: Remove the print commands

**Review by Rishit02 — COMMENTED** [review]
LGTM. Minor nitpicks

**Comment by pjanthony2001** [comment]
LGTM

**Pull request merged**